### PR TITLE
Hds 1850 Header navigation sub menu arrow fix

### PR DESCRIPTION
**Pull request by minnas**
## Description

This pr fixes closed sub-menu arrow direction based on Figma's design

## Motivation and Context

[HDS-1850](https://helsinkisolutionoffice.atlassian.net/browse/HDS-1850)

## How Has This Been Tested?

Tested locally
(Browsers: Firefox, Chrome, Safari)

## Demo

[Storybook](https://city-of-helsinki.github.io/hds-demo/header-dropdown-arrow-fix)

## Screenshots (if appropriate):

[HDS-1850]: https://helsinkisolutionoffice.atlassian.net/browse/HDS-1850?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
arrow-right
arrow-left

### CODE REVIEW

**Comment by timwessman** [comment]
works correctly. Maybe `getPosition: () => DropdownMenuPosition` should be `@internal` prop

File: packages/react/src/components/header/components/navigationLink/NavigationLink.tsx [hunk 1]
```diff
@@ -161,6 +169,14 @@ export const NavigationLink = <T extends React.ElementType = 'a'>({
   const { openMainNavIndex, setOpenMainNavIndex } = useHeaderNavigationMenuContext();
   const containerRef = useRef<HTMLSpanElement>(null);
   const isSubNavLink = openSubNavIndex !== undefined && setOpenSubNavIndex !== undefined;
+  const [menuSize, setMenuSize] = useState<DOMRect>(undefined);
+
+  useLayoutEffect(() => {
+    const size = containerRef?.current?.getBoundingClientRect();
+    // eslint-disable-next-line no-console
+    console.log('MENU SIZE', size);
```
Comment (timwessman) [inline packages/react/src/components/header/components/navigationLink/NavigationLink.tsx:177]: is this still needed?

File: packages/react/src/components/header/components/navigationLink/navigationLinkDropdown/NavigationLinkDropdown.tsx [hunk 2]
```diff
@@ -56,6 +56,10 @@ export type NavigationLinkDropdownProps = React.PropsWithChildren<{
    * @internal
    */
   depth: number;
+  /**
+   * Function that return dropdown direction.
+   */
+  getPosition: () => DropdownMenuPosition;
```
Comment (timwessman) [inline packages/react/src/components/header/components/navigationLink/navigationLinkDropdown/NavigationLinkDropdown.tsx:62]: This could be `@internal`

**Review by timwessman — CHANGES_REQUESTED** [review]

File: packages/react/src/components/header/components/navigationLink/NavigationLink.tsx (hunk 1 above)
Comment (minnas) [reply packages/react/src/components/header/components/navigationLink/NavigationLink.tsx]: Removed, added only for testing,

**Review by timwessman — APPROVED** [review]

**Pull request merged**